### PR TITLE
prepare for 2.5

### PR DESCRIPTION
**Pull request by Davka**

### CODE REVIEW

**Comment by BrutalBirdie** [comment]
I will deploy this PR to the dev server tomorrow and test it out.

**Comment by BrutalBirdie** [comment]
Certbot on Ubuntu 20 wants to be installed via snapd.. I nope to that.
Will switch to acme.sh.

**Comment by defnull** [comment]
You can always install and update Certbot via `pip`.
`sudo apt-get install python3-pip; sudo pip install certbot`

File: README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.4/install.html#
 | | `bbb_letsencrypt_enable` | Enable letsencrypt/HTTPS | `yes` |
 | ⚠️ when using letsencrypt| `bbb_letsencrypt_email` | E-mail for use with letsencrypt | | |
 | | `bbb_letsencrypt_api` | Set letsencrypt api | `https://acme-v02.api.letsencrypt.org/directory` | Use this variable to change letsencrypt API URL (example: staging API `https://acme-staging-v02.api.letsencrypt.org/directory`) |
+| | `bbb_letsencrypt_repo` | The github rebo to use for renewal handling - default acme.sh | <https://github.com/acmesh-official/acme.sh.git> | |
```
Comment (EdJoPaTo) [inline README.md:36]:
```suggestion
| | `bbb_letsencrypt_repo` | The github repo to use for renewal handling - default acme.sh | <https://github.com/acmesh-official/acme.sh.git> | |
```

Comment (BrutalBirdie) [reply README.md:36]: Yes!

**Review by BrutalBirdie — APPROVED** [review]

**Pull request merged**